src/services/services.js

```diff
@@ -1,30 +1,31 @@
 // NOTE add any services and api calls here. Don't expose api keys here. Use .env files for that purpose
-const endpoint = 'http://localhost:3000/sampleData.json';
+const api = (endpoint) => {
+  let baseURL = 'http://localhost:3000/';
+  return baseURL + endpoint;
+}
+
+const fetchData = (URL, args) =>{
+  return fetch( api(URL) )
+         .then(response => {
+           return response.json();
+         })
+         .then(data =>{
+           return data;
+         })
+         .catch(err => {
+           console.log("Error: ", err);
+         })
+};
 
 const services = {
```

Comment:
I recommend capitalizing all utilities.

Reply:
Are you referring to the overall utility (E.g: Services), or the individual functions inside of the utility?

Reply:
"Services" instead of "services."